### PR TITLE
Add a prometheus link to allow quick access to prometheus ui.

### PR DESCRIPTION
**Pull request by shokada**

### CODE REVIEW

File: promgen/templates/promgen/project_detail_configuration.html [hunk 1]
```diff
@@ -48,6 +48,8 @@
 
   {% endif %}
 
-  <div class="panel-footer">Shard: <a href="{{project.shard.get_absolute_url}}">{{project.shard.name}}</div>
-
+  <div class="panel-footer">
+    Shard: <a href="{{project.shard.get_absolute_url}}">{{project.shard.name}}</a>
+    (<a href="{{project.shard.url}}">{{project.shard.url}}</a>)
```
Comment (kfdm) [inline promgen/templates/promgen/project_detail_configuration.html]:
```suggestion
    ( <a href="{{project.shard.url}}">{{project.shard.url}}</a> )
```

File: promgen/templates/promgen/service_block_projects.html [hunk 2]
```diff
@@ -7,6 +7,7 @@
     <div class="panel-heading">
         Shard
         <a href="{{shard.get_absolute_url}}">{{shard.name}}</a>
+        (<a href="{{shard.url}}">{{shard.url}}</a>)
```
Comment (kfdm) [inline promgen/templates/promgen/service_block_projects.html]:
```suggestion
        ( <a href="{{shard.url}}">{{shard.url}}</a> )
```

File: promgen/templates/promgen/project_detail_configuration.html (hunk 1 above)
Comment (shokada) [reply promgen/templates/promgen/project_detail_configuration.html]: fixed

File: promgen/templates/promgen/service_block_projects.html (hunk 2 above)
Comment (shokada) [reply promgen/templates/promgen/service_block_projects.html]: fixed

**Pull request merged**